azext_edge/edge/providers/support/mq.py

```diff
@@ -132,13 +98,7 @@ def fetch_services():
 def fetch_replicasets():
     processed = process_replicasets(
```

Comment:
nit: should we directly return the process_replicasets when there is only one function call?